tests/rptest/scale_tests/partition_balancer_scale_test.py

```diff
@@ -144,3 +146,128 @@ def all_reconfigurations_done():
         wait_until(all_reconfigurations_done, 300, 5)
 
         self.verify()
+
+    @cluster(num_nodes=6)
+    @parametrize(type=MANY_PARTITIONS)
+    @parametrize(type=BIG_PARTITIONS)
+    def test_node_operations_at_scale(self, type):
+        replication_factor = 3
+        if not self.redpanda.dedicated_nodes:
+            # Mini mode, for developers working on the test on their workstation.
+            # (not for use in CI)
+            message_size = 16384
+            message_cnt = 64000
+            consumers = 1
+            partitions_count = 40
+            max_concurrent_moves = 5
+        elif type == self.MANY_PARTITIONS:
+            message_size = 128 * (2 ^ 10)
+            message_cnt = 2000000
+            consumers = 8
+            partitions_count = 18000
+            max_concurrent_moves = 400
+            timeout = 300
+        else:
+            message_size = 512 * (2 ^ 10)
+            message_cnt = 5000000
+            consumers = 8
+            partitions_count = 200
+            max_concurrent_moves = 200
+            timeout = 300
+
+        # set max number of concurrent moves
+        self.redpanda.set_cluster_config(
+            {"partition_autobalancing_concurrent_moves": max_concurrent_moves})
+
+        topic = TopicSpec(partition_count=partitions_count,
+                          replication_factor=replication_factor)
+        self.client().create_topic(topic)
+
+        self._start_producer(topic.name, message_cnt, message_size)
+        self._start_consumer(topic.name, message_size, consumers=consumers)
+        mb = 1024 * 1024
+        self.logger.info(
+            f"waiting for {(message_size*message_cnt) / mb} MB to be produced to "
+            f"{partitions_count} partitions ({((message_size*message_cnt) / mb) / partitions_count} MB per partition"
+        )
+        # wait for the partitions to be filled with data
+        self.producer.wait_for_acks(message_cnt // 2,
+                                    timeout_sec=timeout,
+                                    backoff_sec=5)
+
+        admin = Admin(self.redpanda)
+        brokers = admin.get_brokers()
+        # decommission one of the nodes
+        decommissioned_id = random.choice(brokers)['node_id']
+        self.logger.info(
+            f"cluster brokers: {brokers}, decommissioning: {decommissioned_id}"
+        )
+        admin.decommission_broker(decommissioned_id)
+
+        def decommission_ended():
+            replicas = self.node_replicas([topic.name], decommissioned_id)
+            self.logger.debug(
+                f"decommissioned node {decommissioned_id} hosts {len(replicas)} replicas"
+            )
+            if len(replicas) != 0:
+                return False
+            all_brokers = set()
+            for n in self.redpanda.nodes:
+                current_id = self.redpanda.node_id(n)
```

Comment:
I think references to `node_id()` need to be swapped for `idx()` calls

Reply:
Looks like we just removed the test altogether. Any other way to validate this is doing the right thing?

Reply:
>i removed the test as it would require ton of backports

LGTM. Approved, pending CI